dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/EventOperationParamsMapper.java

```diff
@@ -112,14 +112,15 @@ public EventSearchParams map(EventOperationParams operationParams)
     ProgramStage programStage = validateProgramStage(operationParams.getProgramStageUid());
     OrganisationUnit requestedOrgUnit = validateRequestedOrgUnit(operationParams.getOrgUnitUid());
 
-    validateOrgUnitMode(operationParams, user, program);
-    OrganisationUnitSelectionMode orgUnitMode =
-        getOrgUnitMode(requestedOrgUnit, operationParams.getOrgUnitMode());
+    if (operationParams.getOrgUnitMode() != null) {
```

Comment:
So far most validation functions get the `parameter` passed in regardless of whether its null or not. They then start with a guard clause exiting when there is nothing to validate. This particular function might already do a null check.

This makes the code in the main mapper function more readable IMHO.

Reply:
Right. This check will be done in the web layer from now on, so here won't be necessary anymore because it can't be null at this point.